### DISCUSSION TITLE
ADD: Maybe Finance - Personal Finance

### DISCUSSION THREAD

**Discussion by SergiSvK** [coollabsio/coolify#4077] [Service Template Requests]
Hi everyone 👋,

I would like to share a very promising open-source personal finance project. Although the project is still in its early stages, it has great potential. I have adapted the Docker Compose code to work in Coolify without major difficulties.

My proposal is to gauge the community's interest in this project. If there is enough interest, we could consider officially integrating it into Coolify.

🆕 Official repository: https://github.com/maybe-finance/maybe
![brave_zYYKPnnUVT](https://maybe.co/assets/screenshot-65d03882.png)

👨🏼‍💻 My Docker Compose: https://gist.github.com/SergiSvK/660d2c14891a269c194084a4b70876ca 
![brave_zYYKPnnUVT](https://github.com/user-attachments/assets/aba6d358-8d07-4774-b4dd-ce224eed1a34)

**Comment by albertorizzi** [comment]
I created a PR to add Maybe finance as service: https://github.com/coollabsio/coolify/pull/4375

**Comment by JunekCZ** [comment]
Hey, I tried to install it, but every time I fill in the registration, it fails and draws an error. :( Something in the Coolify update may have broken it, maybe?

**Reply by albertorizzi** [reply]
Maybe has released a new version of Docker Compose,so the Docker Compose template inside Coolify needs to be updated.
